src/main/java/frc/robot/subsystems/indexer/Indexer.java

```diff
@@ -336,12 +337,13 @@ else if (rejectionTimerStarted && rejectionTimer.hasElapsed(Constants.INDEXER_SH
                     shootBall();
                     intakeBall();
                     rejectionTimerStarted = false;
-                    state = State.INTAKE1;
+                    state = State.FIELD2;
                 }
+                /* This shouldn't happen, it would be at a state with 2 balls not 1
```

Comment:
This can happen, 1 ball tower, 1 ball intake

Reply:
True